docs/source/howto/cli.rst

```diff
@@ -118,3 +118,23 @@ You can enable detailed logs by passing the ``--debug`` option, or by setting th
    geoextent --debug -b -t muenster_ring_zeit.geojson
 
    GEOEXTENT_DEBUG=1 geoextent -b -t muenster_ring_zeit.geojson
+
+Details
+^^^^^^^
+You can enable details for folders and ZIP files by passing the ``--details`` option, this option allows you to access
+to the geoextent of the individual files inside the folders/ ZIP files used to compute the total bounding box (bbox)
```

Comment:
"total bounding box" sounds a bit off - what do you thinkg about "aggregated" or "merged" ?